hydra/core/default_element.py

```diff
@@ -116,19 +117,20 @@ def set_package_header(self, package_header: Optional[str]) -> None:
         if package_header is None:
             return
 
-        if "_group_" in package_header or "_name_" in package_header:
-            path = self.get_config_path()
-            url = "https://hydra.cc/docs/next/upgrades/1.0_to_1.1/changes_to_package_header"
-            deprecation_warning(
-                message=dedent(
-                    f"""\
-                    In '{path}': Usage of deprecated keyword in package header '# @package {package_header}'.
-                    See {url} for more information"""
-                ),
-            )
-
-        if package_header == "_group_":
-            return
+        if not version.base_at_least("1.2"):
+            if "_group_" in package_header or "_name_" in package_header:
+                path = self.get_config_path()
+                url = "https://hydra.cc/docs/next/upgrades/1.0_to_1.1/changes_to_package_header"
+                deprecation_warning(
+                    message=dedent(
+                        f"""\
+                        In '{path}': Usage of deprecated keyword in package header '# @package {package_header}'.
+                        See {url} for more information"""
+                    ),
+                )
+
+            if package_header == "_group_":
+                return
```

Comment:
hmm not sure if I missed anything here...  it seems like this PR only removes the warning but doesn't actually remove the support?

Reply:
Well the special "_group_" handling is only for < "1.2",
but you're right, I need to avoid the "_group_" handling on line 143 also